taylorseager/CulinaryChronicles
# Nav bar updates

## Description

- Removed noAuthNavBar
- Changed name of NavBarAuth to NavBar
- Updated necessary imports and names of NavBar
- Updated _app.js - had TODOs from template based on auth/no auth
- Updated index.js - had TODOs from template based on auth/no auth
- Home page updated to include sign out button
- "Home" button removed and application title made home route

## Related Issue
#22 #28 #8 

## Motivation and Context
This will make the user experience more enjoyable and flow through the application smoother.

## How Can This Be Tested?
Pull down repo -> 
fetch & checkout branch (navBarUpdates) ->
Click through the nav bar to see if the routes work as expected

## Screenshots (if appropriate):
![Screenshot 2024-05-20 at 8 34 08 PM](https://github.com/taylorseager/CulinaryChronicles/assets/154686027/7c1c1ee4-9761-47fe-bb21-716260d69246)

## Types of changes
